### PR TITLE
Change the code structure

### PR DESCRIPTION
**Pull request by knicked**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,34 +1,35 @@
...
-    let ver = 1
-    return {
-      'ver': ver + 1,
-      'prevHash': prevHash,
-      'transactions': transactions,
-      'proof': proof,
-      'timestamp': Date.now()
-    }
+  constructor (prevHash, transactions, proof) {
+    this.ver = 1
```
Comment (xdk78) [inline src/index.js]: nice, but we need increment this by 1

Comment (knicked) [reply src/index.js]: Should we increment this by 1 every block? Why?

Comment (xdk78) [reply src/index.js]: every block must have own unique 'index' too

Comment (knicked) [reply src/index.js]: Shouldn't `ver` be referring to the version of the protocole, not to the block's index?

Comment (xdk78) [reply src/index.js]: ok, just add one more field `index` and increment this one by 1

Comment (knicked) [reply src/index.js]: Done

**Pull request merged**